cogs/strike.py

```diff
@@ -161,7 +161,7 @@ async def no_manual_moderation_action_button_callback(self, _: discord.Button, i
         the manual moderation tracker subroutine that sent the view,
         so all that is required is to delete the original message that sent this view.
         """
-        logger.debug("\"No\" button pressed. %s", interaction)
+        logger.debug('"No" button pressed. %s', interaction)
```

Comment:
Prefer strings to use double quotes & escape interior double quotes rather than using single quotes